node/src/sub_lib/wallet.rs

```diff
@@ -128,7 +128,7 @@ impl Wallet {
             WalletKind::Address(address) => H160(address.0),
             WalletKind::PublicKey(public) => H160(*public.address()),
             WalletKind::SecretKey(key_provider) => key_provider.address(),
-            WalletKind::Uninitialized => panic!("No address for an uninitialized wallet!"),
+            WalletKind::Uninitialized => panic!("No address for an uninitialized wallet!"), // TODO: If we can get rid of it, it'll be awesome!
```

Comment:
Have you tried out guys? Not saying that you necessarily should.

Reply:
We have created a card for this task;
https://github.com/orgs/MASQ-Project/projects/8/views/1?pane=issue&itemId=90819582&issue=MASQ-Project%7CMASQ-Node-issues%7C816

Trying to keep things simple if we can :)

Reply:
Ok. Valid.